src/components/textarea/textarea.component.spec.ts

```diff
@@ -515,25 +515,6 @@ describe('Textarea', () => {
             )
         );
 
```

Comment:
why u no let unit test in place? (or write a new one) @SiNEXiST

Reply:
He removed it because the height calculation depends on the width of the textarea. Since Angular doesn't load the styles during the tests, the width seems unpredictable.

Reply:
Don't be afraid to set inline styles (yes, this atrocity is acceptable in unit tests) in tests where the outcome depends on them :grinning:

Reply:
The problem is not actually setting the inline style on the gtx-textarea, but the HTML textarea element needs to have some styles applied to take the width of the parent into account (see https://stackoverflow.com/questions/3165083/how-to-make-textarea-to-fill-div-block ).